it/v5/AuthISpec.scala

```diff
@@ -62,22 +62,23 @@ class AuthISpec extends IntegrationBaseSpec {
 
   "Calling the sample endpoint" when {
 
-    "the NINO cannot be converted to a MTD ID" should {
+    "MTD ID lookup fails with a 500" should {
 
       "return 500" in new Test {
         override val nino: String = "AA123456A"
 
         override def setupStubs(): StubMapping = {
           AuditStub.audit()
-          MtdIdLookupStub.internalServerError(nino)
+          MtdIdLookupStub.error(nino, Status.INTERNAL_SERVER_ERROR)
         }
 
         val response: WSResponse = await(request().post(Json.parse(requestJson)))
         response.status shouldBe Status.INTERNAL_SERVER_ERROR
       }
     }
+  }
```

Comment:
This needs to include the "MTD ID lookup fails with a 403" case as this is the scenario that this ticket should fix. (It might be instructive to add the test to the code base without your changes and watch it fail.)